themes/classic/templates/checkout/_partials/cart-detailed-product-line.tpl

```diff
@@ -66,7 +66,7 @@
     <br/>
 
     {foreach from=$product.attributes key="attribute" item="value"}
-      <div class="product-line-info">
+      <div id={$attribute} class="product-line-info">
```

Comment:
```suggestion
      <div class="product-line-info attribute-{$attribute|lower}-info">
```

Reply:
You should use class instead, to avoid duplicated ids

Reply:
Done thx :+1: